### PR TITLE
4815 Users dashboard

### PR DESCRIPTION
**Pull request by RemcoSimonides**
Added dashboard for Users

### CODE REVIEW

File: env/env.blockframes-remco.ts [hunk 1]
```diff
@@ -148,7 +148,8 @@ export const bigQueryAnalyticsTable = 'blockframes-remco.analytics_234801912.eve
 // ========
 
 export const datastudio = {
```
Comment (phpgeekfr) [inline env/env.blockframes-remco.ts:150]: you are not updating every env files ?

File: libs/admin/src/lib/admin-panel/components/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<ng-container *ngIf="!!dashboardUrl">
+  <section fxLayout fxLayoutAlign="center">
+    <iframe
```
Comment (phpgeekfr) [inline libs/admin/src/lib/admin-panel/components/dashboard/dashboard.component.html:3]: they don't provide anything else than iframe ?

**Review by phpgeekfr — APPROVED** [review]
LGTM

File: env/env.blockframes-remco.ts (hunk 1 above)
Comment (RemcoSimonides) [reply env/env.blockframes-remco.ts:150]: No, I only make a dashboard for Staging and Production. I added the staging dashboard to my env.ts so I can see it.

File: libs/admin/src/lib/admin-panel/components/dashboard/dashboard.component.html (hunk 2 above)
Comment (RemcoSimonides) [reply libs/admin/src/lib/admin-panel/components/dashboard/dashboard.component.html:3]: No :/

**Pull request merged**